docs/tasks.md

```diff
@@ -452,4 +452,74 @@ def run_snowflake_queries(*args):
                       'sf_cluster_keys':''}
   )
   uc_to_sf_copy.execute()
-```
\ No newline at end of file
+```
+
+#### Tableau Refresh Operators
+Connect to the Tableau server and trigger the refresh of the data sources or workbooks.
+
+Tableau client uses object GUIDs  to identify objects on the server. At the same time the server does not
+enforce unique names for the objects across the server. This means that multiple objects, e.g. data sources, with the 
+same name can exist on the server. 
+
+To overcome this, operators are using the combination of `project` and `parent_project` parameters to uniquely
+identify the project that owns data source or workbook on the server. Successfull project resolution will be indicated
+in the logs as follows:
+```
+INFO - Querying all projects on site
+
+Parent project identified:
+	Name: My Parent Project
+	ID: 2e14e111-036f-409e-b536-fb515ee534b9
```

Comment:
Is this like the real project_id or for this documentation - mocked up one?

Reply:
This is taken from the real project and few characters have been changed. I can generate fresh GUID if necessary...